src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -170,6 +170,18 @@ public function thereIsCustomerThatPlacedOrder(CustomerInterface $customer, $ord
     }
 
     /**
+     * @Given a customer :customer added something to cart
+     */
+    public function customerStartedCheckout(CustomerInterface $customer)
```

Comment:
Maybe renaming step to `Given a customer has a cart assigned` or something similar